dev-support/cicd/README.md

```diff
@@ -21,25 +21,30 @@ To use them more easily, we provide a Docker image to help committer to handle t
 ## Docker mode
 
 ```
-cd <path-to-submarine-home>/dev-support/cicd
-docker build -t submarine-cicd .
-docker run -it --rm submarine-cicd
+./build_and_start_cicd_image.sh
 ```
 
 Or
 
 ```
-./build_and_start_cicd_image.sh
+cd <path-to-submarine-home>/dev-support/cicd
+docker build -t submarine-cicd .
+docker run -it --rm submarine-cicd
 ```
 
+Apache jira username and password are required in the docker container. You can
+add them to the docker startup command.
+
 And you'll see output like below and then you can decide what to accomplish.
 ```
-$ docker run -it --rm submarine-cicd
+$ docker run -it -e JIRA_USERNAME='username' -e JIRA_PASSWORD='password' --rm submarine-cicd 
```

Comment:
it can also set these environment variable "-e APACHE_ID=aid -e APACHE_NAME="Zhankun Tang""

Reply:
Done